fix/webs.lua

```diff
@@ -0,0 +1,60 @@
+--Cause floating webs to fall
+--By Bumber
+--@module = true
+--[====[
+
+fix/webs
+========
+Turns floating webs into projectiles, causing them to fall down to a valid surface
+Use ``fix/webs -all`` to turn all webs into projectiles, causing webs to fall out of branches, etc.
+
+]====]
+local utils = require "utils"
+
+function fix_webs(air_only)
+    if not dfhack.isMapLoaded() then
+        qerror("Error: Map not loaded!")
+    end
+    
+    local count = 0
+    for i, item in ipairs(df.global.world.items.all) do
+        if item:getType() == df.item_type.THREAD and
+        item.flags.spider_web and
+        item.flags.on_ground and
+        not item.flags.in_job then
+            local valid_tile = true
+            
+            if air_only then
+                local tt = dfhack.maps.getTileBlock(item.pos).tiletype[item.pos.x%16][item.pos.y%16]
```

Comment:
dfhack.maps.getTileType(item.pos) might be easier to use here. See, for example, https://github.com/DFHack/scripts/blob/55ac6f7269a10a1b3e525da2c3f117675b529289/internal/quickfort/dig.lua#L542